common/djangoapps/student/management/commands/force_send_notification_digest.py

```diff
@@ -0,0 +1,155 @@
+        preference_name = const.NOTIFICATION_DAILY_DIGEST_PREFERENCE_NAME
+        subject = const.NOTIFICATION_DAILY_DIGEST_SUBJECT
+        from_email = const.NOTIFICATION_EMAIL_FROM_ADDRESS
+
+        if namespace == "All":
+            digests_sent = send_notifications_digest(from_timestamp, to_timestamp, preference_name, subject, from_email)
+        else:
+            digests_sent = send_notifications_namespace_digest(
+                namespace, from_timestamp, to_timestamp, preference_name, subject, from_email
+            )
+        return digests_sent
+
+    def send_weekly_digest(self, namespace='All'):
+        """
+        Sends the weekly digest.
+        """
+
+        if const.NOTIFICATION_DIGEST_SEND_TIMEFILTERED:
+            from_timestamp = datetime.datetime.now(pytz.UTC) - datetime.timedelta(days=7)
+        else:
+            from_timestamp = None
+
+        to_timestamp = datetime.datetime.now(pytz.UTC)
+        preference_name = const.NOTIFICATION_WEEKLY_DIGEST_PREFERENCE_NAME
+        subject = const.NOTIFICATION_WEEKLY_DIGEST_SUBJECT
+        from_email = const.NOTIFICATION_EMAIL_FROM_ADDRESS
+
+        if namespace == "All":
+            digests_sent = send_notifications_digest(from_timestamp, to_timestamp, preference_name, subject, from_email)
+        else:
+            digests_sent = send_notifications_namespace_digest(
+                namespace, from_timestamp, to_timestamp, preference_name, subject, from_email
+            )
+        return digests_sent
+
+    def handle(self, *args, **options):
+        """
+        Management command entry point, simply call into the send_notifications_digest or the
+        send_notifications_namespace_digest depending on the passed the parameters.
+        The expected command line arguments are:
+         --daily: Sends the daily digest.
+         --weekly: Sends the weekly digest.
+         --ns=NAMESPACE : Sends the notifications for the particular NAMESPACE.
+        """
+
+
+        if not settings.FEATURES.get('ENABLE_NOTIFICATIONS', False):
+            print 'ENABLE_NOTIFICATIONS not set to "true". Stopping...'
+            return
+
+        usage = "usage: %prog [--daily] [--weekly] [--ns=NAMESPACE]"
+        parser = OptionParser(usage=usage)
+
+        log.info("Running management command ...")
+
+        if options['send_daily_digest']:
+            log.info("Sending the daily digest with namespace=%s...", options['namespace'])
+            weekly_digests_sent = self.send_daily_digest(options['namespace'])
+            log.info("Successfully sent %s digests...", weekly_digests_sent)
+
+        if options['send_weekly_digest']:
+            log.info("Sending the weekly digest with namespace=%s...", options['namespace'])
+            daily_digests_sent = self.send_weekly_digest(options['namespace'])
+            log.info("Successfully sent %s digests...", daily_digests_sent)
+
+        if not options['send_weekly_digest'] and not options['send_daily_digest']:
```

Comment:
Yes, it possible based on the command line args options. In general, this command line django admin command really is used for testing purposes, it's not how digests are actually triggered in production (which uses a timer mechanism in edx-notifications).